TACC/Core-Portal
# task/TV3-70: Apps Tray

## Overview
A working Apps Tray for Tapis v3.

- Simplified `AppTrayEntry` models
- Simplified `AppsTrayView` methods

## Related

* [TV3-70](https://jira.tacc.utexas.edu/browse/TV3-70)

## Testing

1. Run migrations
2. Go to https://cep.test/core/admin/workspace/apptraycategory/ and add a category
3. Go to https://cep.test/core/admin/workspace/apptrayentry/ and add some app entries. The `hello-world` app in the a2cps tenant is public, so use that one. You can add it multiple times to make sure it still shows up on the frontend in multiple ways. See UI Photos.
4. Go to https://cep.test/workbench/applications and confirm your entries appear as expected

## UI Photos
![Screen Shot 2022-10-14 at 2 41 43 PM](https://user-images.githubusercontent.com/20326896/195929568-36c9df9d-ba4e-44be-9558-fb525a6de986.png)
![Screen Shot 2022-10-14 at 2 40 51 PM](https://user-images.githubusercontent.com/20326896/195929569-4d9ebb1d-5e01-4591-aea3-6d3c7c725099.png)
![Screen Shot 2022-10-14 at 2 40 38 PM](https://user-images.githubusercontent.com/20326896/195929570-91d0c142-36d4-43d6-88f2-ed41e12ff3eb.png)
![Screen Shot 2022-10-14 at 2 40 30 PM](https://user-images.githubusercontent.com/20326896/195929571-c3761362-6c50-405a-9496-42dd0cd39440.png)
![Screen Shot 2022-10-14 at 2 40 13 PM](https://user-images.githubusercontent.com/20326896/195929572-9f206756-4139-4ceb-af5f-ba6b5a8f455b.png)
